ci/jenkins/mellanox-ci/common/clean_common.sh

```diff
@@ -0,0 +1,170 @@
+#!/bin/bash -x
```

Comment:
Since the path (`ci/jenkins/mellanox-ci`) already includes "ci", how about changing it to `ci/jenkins/mellanox`?

Reply:
Done